src/extension/content-script/batteries/VimeoVideo.ts

```diff
@@ -0,0 +1,48 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/vimeo.com\/.*/;
+
+const battery = (): void => {
+  document.getElementsByClassName("first")[0].innerHTML;
+  const videoDescription = document.querySelector(".first");
+  const channelLink = document.querySelector(".js-user_link");
```

Comment:
I think this query matches also the ones from the "similar videos" on the side? 
Is it save that this is the link to the current channel?

maybe we can additionally add `.clip_info-subline--watch`
or additionally even check that `data-fatal-attraction` attribute on the link? 

we just need to be save that it is the name of the channel :)

Reply:
I'll look into this but I wasn't sure what channelLink is used for. I assumed it's just to identify that we're on a video page. So I was thinking just the confirmation that it exists would be enough.

Reply:
I've changed to `.clip_info-subline--watch` I agree this makes more sense. I don't think `data-fatal-attraction` helps though.

Reply:
👍 
(the name is extracted from that link and that name is shown in Alby then)